deploy/charts/arcadia/values.yaml

```diff
@@ -122,17 +122,21 @@ dataprocess:
   enabled: true
   image: kubeagi/data-processing:v0.1.0
   port: 28888
-  env:
-    minio:
-      secure: "False"
-      apiURL: arcadia-minio:9000
-    knowledge:
```

Comment:
@wangxinbiao  You can hard code this default chunksize and chunk overlap in your code as constant values. 

And allow users to configure these parameters for each data processing task